mmv1/third_party/terraform/services/vertexai/resource_vertex_ai_feature_online_store_test.go

```diff
@@ -0,0 +1,88 @@
+package vertexai_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccVertexAIFeatureOnlineStore_vertexAiFeatureonlinestoreWithBigtable_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVertexAIFeatureOnlineStore_vertexAiFeatureonlinestoreWithBigtable_basic(context),
+			},
+			{
+				ResourceName:            "google_vertex_ai_feature_online_store.featureonlinestore_bigtable",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "etag", "region", "force_destroy", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccVertexAIFeatureOnlineStore_vertexAiFeatureonlinestoreWithBigtableExample_update(context),
+			},
+			{
+
+				ResourceName:            "google_vertex_ai_feature_online_store.featureonlinestore_bigtable",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name", "etag", "region", "force_destroy", "labels", "terraform_labels"},
+			},
+		},
+	})
+}
+
+func testAccVertexAIFeatureOnlineStore_vertexAiFeatureonlinestoreWithBigtable_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+##FeatureOnlineStore With BigTable
+resource "google_vertex_ai_feature_online_store" "featureonlinestore_bigtable" {
+  name     = "terraform2%{random_suffix}"
```

Comment:
Could you please update this to have the prefix `tf-test-`? This helps us to automatically identify and delete resources that are created by acceptance tests but not deleted correctly when the test finishes.

Reply:
Done